angular/README.md

```diff
@@ -1,10 +1,25 @@
-# MyApp
+# WxCC Widget Starter - Angular
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 14.2.11.
 
-## Development server
+This widget starter can be run on `localhost` to be reviewed. In case you (the developer) are using this widget starter as a foundation of your custom widget, running the sandbox environment on `localhost` would be a perfect way to build your widget's logic and interface.
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
+**Note**: please keep in mind that we recommend to design every widget as responsive. With the exception of widgets for Agent or Supervisor Desktop **header** zone (where widgets should not exceed 64px height), you can assume that your custom widget might end up getting placed on various canvas sizes. Based on the Agent's screen size and configuration by the Administrator, it could even end up with a different aspect ratio. Please keep that in mind while testing your responsive widget's behavior.
+
+# Development
+
```

Comment:
I would add in some blurb in this section to let them know if they want to make edits to the widget they can do so in the /my-custom-component folder